gh-110937: Document full public importlib.metadata.Distribution API (#143480)
python/cpython: Doc/library/importlib.metadata.rst

Distributions
=============

While the module level API described above is the most common and convenient usage,

> you can get all of that information from the :class:`Distribution` class.

Reviewer: nit: I prefer imperative voice.

```suggestion
all that information is accessible from the :class:`Distribution` class.
```

:class:`!Distribution` is an abstract object that represents the metadata for
a Python `Distribution Package <https://packaging.python.org/en/latest/glossary/#term-Distribution-Package>`_.

> You can get the concrete :class:`!Distribution` subclass instance for an installed

Reviewer: ```suggestion
Get the concrete :class:`!Distribution` subclass instance for an installed
```
